src/component.coffee

```diff
@@ -390,15 +392,13 @@ Ember.AddeparMixins.ResizeHandlerMixin,
 
   # rows that were selected directly or as part of a previous
   # range selection (shift-click)
-  persistedSelection: Ember.computed ->
-    Ember.ArrayProxy.createWithMixins Ember.MutableArray, {content:[]}
+  persistedSelection: Ember.computed -> []
```

Comment:
One thing I'm going to do with ember-widgets is turn off prototype extensions for gh_pages so that people are forced to support it.